### PR TITLE
[#75] Fixed problems on site with HTML

### PR DESCRIPTION
**Pull request by OleksSobol**

### CODE REVIEW

File: _includes/.fuse_hidden0000034f00000001 [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<!-- <nav class="nav"> -->
```
Comment (khusnetdinov) [inline _includes/.fuse_hidden0000034f00000001]: What does this file do?

File: _sass/_base.scss [hunk 2]
```diff
@@ -20,6 +20,12 @@
     }
   }
 
+  .yandex-metrika-counter {
```
Comment (khusnetdinov) [inline _sass/_base.scss]: Why do you add this? What problem solve this styles code?

Comment (OleksSobol) [reply _sass/_base.scss]: Your webpage is using inline CSS styles! One of the problem from https://seositecheckup.com/seo-audit/betterdocs.herokuapp.com#common-seo-issues

**Comment by khusnetdinov** [comment]
@OleksSobol, you need first update branch and after place commit. With you case frist solution and after merge master can ve problems. Please use rebase or fetch and merge updates to work branches before commit.

**Comment by OleksSobol** [comment]
Here is app with my changes: [ruby](https://ruby-fundamental-copy.herokuapp.com/gotchas). 
Do not have any errors by [this website](https://validator.w3.org/nu/?showsource=yes&showoutline=yes&showimagereport=yes&doc=https%3A%2F%2Fruby-fundamental-copy.herokuapp.com%2F);
Score went up for 4 points [http://www.seowebpageanalyzer.com/](http://www.seowebpageanalyzer.com/)

File: Gemfile [hunk 3]
```diff
@@ -13,4 +13,5 @@ gem 'scss_lint'
 
 group :jekyll_plugins do
   gem 'jekyll-multiple-languages-plugin'
+  gem 'jekyll-sitemap'
```
Comment (khusnetdinov) [inline Gemfile]: @OleksSobol Probably you updated you branch not ptoblerly, you got files from previous PR https://github.com/howtohireme/ruby.fundamental/commit/94feecc55ed7ab5a636baa0a4cf6bc4be4cb43ca Check it please.

**Comment by OleksSobol** [comment]
Hi! That's should be it!

**Comment by OleksSobol** [comment]
That's should be it! And I also deployed my branch to heroku. You can look at it [here](https://ruby-copy.herokuapp.com/).

File: _layouts/default.html [hunk 4]
```diff
@@ -1,9 +1,10 @@
 <!DOCTYPE html>
-<html>
+<html lang="{{ site.lang}}">
```
Comment (khusnetdinov) [inline _layouts/default.html]: Space `lang="{{ site.lang }}`

File: _layouts/default.html [hunk 5]
```diff
@@ -1,9 +1,10 @@
...
   <head>
     {% include head.html %}
   </head>
-  <body class="l-body">
+  <!-- <body class="l-body"> -->
```
Comment (khusnetdinov) [inline _layouts/default.html]: remove `<!-- <body class="l-body"> -->`

**Comment by OleksSobol** [comment]
Check it now.

**Pull request merged**